src/databricks/labs/remorph/install.py

```diff
@@ -487,23 +541,39 @@ def run(
         return config
 
     @classmethod
-    def install_bladerunner(cls):
+    def install_bladerunner(cls, artifact: Path | None = None):
         local_name = "bladerunner"
         pypi_name = "databricks-bb-plugin"
-        TranspilerInstaller.install_from_pypi(local_name, pypi_name)
+        TranspilerInstaller.install_from_pypi(local_name, pypi_name, artifact)
 
     @classmethod
-    def install_morpheus(cls):
+    def install_morpheus(cls, artifact: Path | None = None):
         java_version = cls.get_java_version()
-        if java_version < 110:
+        if java_version is None or java_version < 110:
             logger.warning(
                 "This software requires Java 11 or above. Please install Java and re-run 'install-transpile'."
             )
             return
         product_name = "databricks-morph-plugin"
         group_id = "com.databricks.labs"
         artifact_id = product_name
-        TranspilerInstaller.install_from_maven(product_name, group_id, artifact_id)
+        TranspilerInstaller.install_from_maven(product_name, group_id, artifact_id, artifact)
+
+    @classmethod
+    def install_artifact(cls, artifact: str):
+        path = Path(artifact)
+        if not path.exists():
+            logger.error(f"Could not locate artifact {artifact}")
+            return
+        if "morpheus-lsp" in path.name:
+            cls.install_morpheus(path)
+        elif "databricks-bb-plugin" in path.name:
+            cls.install_bladerunner(path)
+        # don't automatically install RCT, but keep the ability to install it manually
+        elif "remorph-community-transpiler" in path.name:
+            local_name = "remorph-community-transpiler"
+            pypi_name = f"databricks-labs-{local_name}"
```

Comment:
Please consider this:
```suggestion
        elif "remorph-community-transpiler" in path.name:
            local_name = "remorph-community-transpiler"
            pypi_name = f"databricks-labs-{local_name}"
        else:
            logger.fatal(f"Cannot install unsupported artifact: {artifact}")
```
(Without this it is impossible to know that an installation is silently being ignored.)

Reply:
Done